packages/flutter/test/rendering/object_test.dart

```diff
@@ -377,6 +377,12 @@ void main() {
     root.buildScene(ui.SceneBuilder()).dispose();
     expect(calledBack, true);
   });
+
+  test('ContainerParentDataMixin asserts parentData type', () {
+    final TestRenderObject renderObject = TestRenderObjectWithoutSetupParentData();
+    final TestRenderObject child = TestRenderObject();
+    expect(()=>renderObject.add(child),throwsAssertionError);
```

Comment:
```suggestion
    expect(() => renderObject.add(child), throwsAssertionError);
```

The formatting is wrong here.